src/PortingAssistant.Client.Client/Reports/ReportExporter.cs

```diff
@@ -122,10 +122,7 @@ private async Task<bool> WriteReportToFileAsync<T>(T obj, string FilePath)
         {
             try
             {
-                using var destinationStream = new FileStream(FilePath, FileMode.Create);
-                using var memoStream = new MemoryStream(obj.Serialize<T>());
```

Comment:
Since we are removing this we can remove the DataExtensions static class below too.

Reply:
Yes, if you don't intend to use it later.